<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Category Filter Tester</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="simQt::CategoryDataBreadcrumbs" name="breadcrumbs" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="simQt::CategoryFilterWidget" name="categoryFilterWidget" native="true"/>
     </item>
     <item>
      <widget class="simQt::CategoryFilterWidget2" name="categoryFilterWidget2" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="smallButton">
       <property name="text">
        <string>Add Small</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="massiveButton">
       <property name="text">
        <string>Add Massive</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="togglePushButton">
       <property name="text">
        <string>Toggle State</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>simQt::CategoryDataBreadcrumbs</class>
   <extends>QWidget</extends>
   <header>simQt/CategoryDataBreadcrumbs.h</header>
  </customwidget>
  <customwidget>
   <class>simQt::CategoryFilterWidget</class>
   <extends>QWidget</extends>
   <header>simQt/CategoryFilterWidget.h</header>
  </customwidget>
  <customwidget>
   <class>simQt::CategoryFilterWidget2</class>
   <extends>QWidget</extends>
   <header>simQt/CategoryTreeModel2.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
